randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -333,31 +344,23 @@ Extra - asset_id: 80
   * Layers: default
   * Normal Door to Green Brinstar Main Shaft/Door to Dachora Room
   > Door to Big Pink
-      Any of the following:
-          Screw Attack or Speed Booster
-          All of the following:
-              Morph Ball
-              Morph Ball Bombs or Power Bombs
+      Screw Attack or Speed Booster or Can Lay Any Bombs
   > Door to Dachora Energy Charge Station
       Speed Booster
 
 > Door to Big Pink; Heals? False
   * Layers: default
   * Normal Door to Big Pink/Door to Dachora Room
   > Door to Green Brinstar Main Shaft
-      Any of the following:
-          Screw Attack or Speed Booster
-          All of the following:
-              Morph Ball
-              Morph Ball Bombs or Power Bombs
+      Screw Attack or Speed Booster or Can Lay Any Bombs
 
 > Door to Dachora Energy Charge Station; Heals? False
   * Layers: default
   * Normal Door to Dachora Energy Charge Station/Door to Dachora Room
   > Door to Big Pink
       Any of the following:
-          # Shinespark up - TODO: double check damage
-          Speed Booster and Shinespark Damage ≥ 120
+          # Shinespark
+          Speed Booster and Shinespark (Beginner) and Shinespark Damage ≥ 120
```

Comment:
wiki lists a way to do the shinespark with less than an etank of damage

Reply:
Added.